examples/glusterfs/README.md

```diff
@@ -0,0 +1,44 @@
+## Glusterfs
+
+[Glusterfs](http://www.gluster.org) is an open source scale-out filesystem. These examples provide information about how to allow Docker containers use Glusterfs volumes.
+
+The examples consist of:
+- A pod that runs on hosts that install Glusterfs client package.
+- A pod that runs on hosts that don't install Glusterfs client package and use [Super Privileged Container](http://developerblog.redhat.com/2014/11/06/introducing-a-super-privileged-container-concept/) to mount Glusterfs.
```

Comment:
We need to roll SPC into the discussion on Security Contexts @pweil- - once paul has basic security contexts running we should look at how we integrate SPC.

----- Original Message -----
> > @@ -0,0 +1,44 @@
> > +## Glusterfs
> > +
> > +[Glusterfs](http://www.gluster.org) is an open source scale-out
> > filesystem. These examples provide information about how to allow Docker
> > containers use Glusterfs volumes.
> > +
> > +The examples consist of:
> > +- A pod that runs on hosts that install Glusterfs client package.
> > +- A pod that runs on hosts that don't install Glusterfs client package and
> > use [Super Privileged
> > Container](http://developerblog.redhat.com/2014/11/06/introducing-a-super-privileged-container-concept/)
> > to mount Glusterfs.
> 
> You are right, this implicit dependency is host (and vendor) specific. The
> SPC helper is provided for convenience for hosts that have SPC running.
> 
> Such dependency *can* be managed by the plugin, but I feel gluster is not the
> only use case requiring a utility SPC. Would love to address this issue in a
> more general framework.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/6174/files#r27481057